internal/template/unstructured.go

```diff
@@ -83,3 +84,31 @@ func applySliceFieldFunc(obj map[string]any, path []string, ff fieldFunc) error
 	}
 	return nil
 }
+
+func processNestedSlice(obj map[string]any, path []string, ff fieldFunc) error {
+	ind := slices.Index(path, "[]")
+	if ind == len(path)-1 {
+		return applySliceFieldFunc(obj, path[:ind], ff)
+	}
+	nestedSlice, found, err := unstructured.NestedSlice(obj, path[:ind]...)
+	// May return error if same key points to map in one place and to string in another
+	if !found || err != nil {
+		return nil
+	}
+	updatedNestedSlice := make([]interface{}, len(nestedSlice))
```

Comment:
Lets use `any` rather then `interface{}` to be consistent with other code

Reply:
done